site/plugins.json

```diff
@@ -431,7 +431,9 @@
     "compatibility": [
       {
         "version": "1.1.5",
-        "siteDependencies": { "next": "<10.0.6" }
+        "siteDependencies": {
+          "next": "<10.0.6"
+        }
```

Comment:
There are two unrelated changes (this one and the final newline), would it be ok to undo them? Thanks!

Reply:
@ehmicky Cool. I have reverted these changes.

Reply:
Thanks!